srcpkgs/python3-curio/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-curio'
+pkgname=python3-curio
+version=1.4
+revision=1
+wrksrc="curio-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+short_desc="Coroutine-based library for concurrent programming using async/await"
+maintainer="Arjan Mossel <arjanmossel@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/dabeaz/curio"
+distfiles="${PYPI_SITE}/c/curio/curio-${version}.tar.gz"
+checksum=57edce81c837f3c2cf42fbb346dee26e537d1659e6605269fb13bd179e068744
+
+do_check() {
+	:
```

Comment:
These packages pass tests with python3-pytest in checkdepends and do_check defined as `PYTHONPATH=$(cd build/lib* && pwd) pytest`, please update.